deploy/015_deploy_celerim_facet.ts

```diff
@@ -0,0 +1,53 @@
+import { HardhatRuntimeEnvironment } from 'hardhat/types'
+import { DeployFunction } from 'hardhat-deploy/types'
+import { ethers, network } from 'hardhat'
+import {
+  diamondContractName,
+  deployFacet,
+  addressesFile,
+  AddressesFile,
+} from './9999_utils'
+import config from '../config/cbridge.json'
+import global from '../config/global.json'
+import fs from 'fs'
+
+interface CBridgeConfig {
+  [network: string]: {
+    cBridge?: string
+    cfUSDC?: string
+    messageBus?: string
+    tokenstoApprove?: string[]
+  }
+}
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  if (!(config as CBridgeConfig)[network.name]) {
+    console.log(`No cBridge config set for ${network.name}. Skipping...`)
+    return
+  }
+
+  const { messageBus } = (config as CBridgeConfig)[network.name]
+  const data = JSON.parse(
+    fs.readFileSync(addressesFile, 'utf8')
+  ) as AddressesFile
+
+  await deployFacet(hre, 'CelerIMFacetMutable', {
```

Comment:
The title of the PR seems confusing as the deploy script is only for the mutable version